pymchelper/readers/shieldhit/reader_bdo2019.py

```diff
@@ -167,7 +166,15 @@ def read_data(self, estimator):
                 if not page.name:
                     page.name = str(page.dettyp)
 
-            estimator.file_format = 'bdo2019'
+            # apply basic normalization for pages with normalisation tag
+            for page in estimator.pages:
+                page_normalisation = getattr(page, 'page_normalized', None)
+                # id 2: X = (sum_j x_j) / (sum_j I_j)          NORMCOUNT, ...
+                if page_normalisation == 2:
```

Comment:
Maybe "2" could be clarified...

```c
/*
   Flags for page->postproc on how to postprocessing the data in page->data.
   Given
   - the data in page->data as x_j
   - for j instances of this simulation
   - which was done with I_j number of paritcles.
   - The resulting data will be termed X and has the units given by page->data

   If this is changed, adopt documentation as well in sh_bdo.h and User's guide.
 */
#define SH_POSTPROC_NONE   0   /* X = x_1                                for GEOMAP type scorers*/
#define SH_POSTPROC_SUM    1   /* X = sum_j x_j                          COUNT, ... */
#define SH_POSTPROC_NORM   2   /* X = (sum_j x_j) / (sum_j I_j)          NORMCOUNT, ... */
#define SH_POSTPROC_AVER   3   /* X = (sum_j x_j * I_j) / (sum_j I_j)    LET, ...*/
```

Reply:
also, I am not sure this will work for all scorers?

Reply:
I see also LET scorers are not correctly implemented yet, this would lead to errors if there are large differences between individual run statistics.
```c
static int _setup_normalize(struct page *p) {

    /* double scorers should not be normalized, since
       these alreay have # of particles implict in the denominator */

    if (p->divide) {
        p->postproc = SH_POSTPROC_AVER;
        return 1;
    }

    switch (p->detector_type) {
    case SH_SDET_NONE:
    case SH_SDET_MATERIAL:
    case SH_SDET_NMEDIUM:
    case SH_SDET_NZONE:
    case SH_SDET_RHO:
        p->postproc = SH_POSTPROC_NONE;
        break;
    case SH_SDET_COUNT:
    case SH_SDET_COUNTPOINT:
        p->postproc = SH_POSTPROC_SUM;
        break;
    default:
        p->postproc = SH_POSTPROC_NORM;
        break;
    }
    return 1;
}
```

Reply:
more work on that (especially averaging in multiple runs) will come in separate PR